app/controllers/api/v1/docs.rb

```diff
@@ -68,10 +70,10 @@ class Docs < Grape::API
         }
         params do
           requires :id, type: Integer, desc: "The ID of the Doc being updated"
-          requires :title, type: String, desc: "The name of the category of articles"
-          requires :category_id, type: Integer, desc: "The category the doc belongs to"
-          requires :body, type: String, desc: "The body/text of the article"
-          requires :user_id, type: Integer, desc: "The author of the article"
+          optional :title, type: String, desc: "The name of the category of articles"
+          optional :category_id, type: Integer, desc: "The category the doc belongs to"
+          optional :body, type: String, desc: "The body/text of the article"
+          optional :user_id, type: Integer, desc: "The author of the article"
           optional :keywords, type: String, desc: "Keywords that will be used for internal search and SEO"
```

Comment:
I am curious why you switched this to optional when they are required by the model?  I will still merge it but just curious.